src/pill/index.ts

```diff
@@ -0,0 +1,34 @@
+import { tag, Tag } from '@storefront/core';
+
+@tag('gb-pill', require('./index.html'))
+class Pill {
+  props: Pill.Props = {
+    selected: false,
+  };
+  refs: { button: HTMLButtonElement };
+
+  onClick(event: MouseEvent & Tag.Event) {
+    event.preventUpdate = true;
+    if (this.props.clickAction) {
+      this.props.clickAction(event);
+    }
+  }
+
+  onClose(event: MouseEvent & Tag.Event) {
+    event.preventUpdate = true;
+    if (this.props.closeAction) {
+      this.props.closeAction(event);
+    }
+  }
+}
+
+interface Pill extends Tag<Pill.Props> { }
+namespace Pill {
+  export interface Props extends Tag.Props {
+    selected: boolean;
+    clickAction?: (event: MouseEvent & Tag.Event) => void;
```

Comment:
Why not name these the same way all of the other callback props are named (`onClick` and `onClose`)?

Reply:
Initially onclick seemed to be a riotjs special tag and so I wanted something to distinguish it.